src/containers/MyNdla/arenaMutations.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { gql } from '@apollo/client';
+import { GQLArenaPageQuery } from '../../graphqlTypes';
+import { useGraphQuery } from '../../util/runQueries';
+
+const arenaPageQueryFragment = gql`
+  fragment ArenaPageQueryFragment on ArenaCategory {
+    __typename
+    description
+    disabled
+    htmlDescription
+    id
+    name
+    postCount
+    slug
+  }
+`;
+
+const topicPageQueryFragment = gql`
+  fragment TopicPageQueryFragment on ArenaTopic {
+    __typename
+    categoryId
+    id
+    locked
+    postCount
+    slug
+    timestamp
+    title
+  }
+`;
+
+export const arenaPageQuery = gql`
+  query arenaPage {
+    arenaCategories {
+      ...ArenaPageQueryFragment
+    }
+  }
+  ${arenaPageQueryFragment}
+`;
+
+export const topicPageQuery = gql`
+  query topicPage {
+    arenaTopics {
+      ...TopicPageQueryFragment
+    }
```

Comment:
Det ser ikkje ut som om vi kan hente topics aleine, men at disse ligger direkte på category. Så du må heller lage en tilsvarende arenaPageQuery og ta med topics der. Om det gav meining.

Reply:
Det trengs et ekstra nivå for å vise en kategori med tråder. 
Forslag til query for å hente en kategori:
```
export const categoryPageQuery = gql`
  query categoryPage($categoryId: Int!, $page: Int!) {
    arenaCategory(categoryId: $categoryId, page: $page) {
      ...ArenaPageQueryFragment
      topics {
        ...TopicPageQueryFragment
      }
      topicCount
    }
  }
  ${arenaPageQueryFragment}
  ${topicPageQueryFragment}
`;
```